### PR TITLE
readme updates

### PR DESCRIPTION
**Pull request by syntrust**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,25 +135,40 @@ cd $GOPATH/src/github.com/QuarkChain/goquarkchain/cmd/cluster
...
-./cluster --cluster_config $CLUSTER_CONFIG_FILE --privkey=$BOOTSTRAP_PRIV_KEY
+cd $GOPATH/src/github.com/QuarkChain/goquarkchain/cmd/cluster
+./cluster --cluster_config ../../tests/testnet/egconfig/cluster_config_template.json --service ${SLAVE_ID}
 ```
-You can read the full boot node URL from the console output in the format: `enode://$BOOTSTRAP_PUB_KEY@$BOOTSTRAP_IP:$BOOTSTRAP_DISCOVERY_PORT`. 
-For example:
+Next, start the `master` service of bootstrap cluster, optionally providing a private key:
 ```bash
-INFO [11-04|18:05:54.832] Started P2P networking  self=enode://011bd77918a523c2d983de2508270420faf6263403a7a7f6daf1212a810537e4d27787e8885d8c696c3445158a75cfe521cfccab9bc25ba5ac6f8aebf60106f1@127.0.0.1:38291
+cd $GOPATH/src/github.com/QuarkChain/goquarkchain/cmd/cluser
```
Comment (pingke) [inline README.md]: remove

File: README.md [hunk 2]
```diff
@@ -135,25 +135,40 @@ cd $GOPATH/src/github.com/QuarkChain/goquarkchain/cmd/cluster
...
+./cluster --cluster_config $CLUSTER_CONFIG_FILE --privkey=$BOOTSTRAP_PRIV_KEY
 ```
+You can read the full boot node URL from the console output in format: `enode://$BOOTSTRAP_PUB_KEY@$BOOTSTRAP_IP:$BOOTSTRAP_DISCOVERY_PORT`.
```
Comment (pingke) [inline README.md]: read -> copy?

File: README.md [hunk 3]
```diff
@@ -85,33 +86,40 @@ Run all the unit tests under `goquarkchain` to verify the environment is correct
 cd $GOPATH/src/github.com/QuarkChain/goquarkchain
 go test ./...
 ```
+
+## Setup Environment Using Docker 
+
+Using [pre-built Docker image](docker/Dockerfile), you will have environment setup to run a cluster inside Docker container.
```
[the remark this answers is not in the document]
Comment (syntrust) [reply README.md]: fixed

File: docker/README.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
+# Quick Start - Use Docker Image to Start a Cluster and Mining
+
+Here we provide detailed instructions for running a cluster and starting the mining process using a pre-built docker image. 
+If you are interested in building everything from scratch, please refer to this [Dockerfile](Dockerfile) for more details.
+
+```bash
+# NOTE the version should be in sync with the release version, e.g. quarkchaindocker/goquarkchain:latest
+$ docker pull quarkchaindocker/goquarkchain:<version tag>
```
[the remark this answers is not in the document]
Comment (syntrust) [reply docker/README.md]: removed

[the remark this answers is not in the document]
Comment (syntrust) [reply docker/README.md]: for later use

File: docker/README.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+                "GAS_LIMIT_ADJUSTMENT_FACTOR": 1024,
+                "GAS_LIMIT_MINIMUM": 5000,
+                "GAS_LIMIT_MAXIMUM": 9223372036854775807,
+                "GAS_LIMIT_USAGE_ADJUSTMENT_NUMERATOR": 3,
+                "GAS_LIMIT_USAGE_ADJUSTMENT_DENOMINATOR": 2
+            },
+...
+```
+
+Note the coinbase address is your Quarkchain wallet address (20-byte recipient + 4-byte full shard key). If you do not have one, please create one from our online testnet / mainnet wallet.
```
[the remark this answers is not in the document]
Comment (syntrust) [reply docker/README.md]: done

File: tests/loadtest/README.md [hunk 6]
```diff
@@ -6,6 +6,13 @@ In order to run loadtest, you need to [run your own GoQuarkChain clusters](../..
 
 A convenient option is to [Use Deploy Tool to Start Clusters](./deployer/README.md#use-deploy-tool-to-start-goquarkchain-clusters).
 
+## Cluster Configuration
+
+- ROOT/CONSENSUS_CONFIG/TARGET_BLOCK_TIME:   foot block interval
```
Comment (pingke) [inline tests/loadtest/README.md]: root block interval

**Pull request merged**